### DISCUSSION TITLE
add note on `template-file` parameter to cloudformation `deploy` must be absolute

### DISCUSSION THREAD

**Discussion by qayshp** [aws/aws-cli#7267] [Q&A] [answered]
Not really a bug or feature request, so discussion seemed like the right place.
I think it would be useful to add a word or two to this about it needing an absolute path, but it has been untouched for years so I'm wondering if anyone agrees.

Relevant line in code:
https://github.com/aws/aws-cli/blob/6c0d168f0b44136fc6175c57c090d4b115437ad1/awscli/customizations/cloudformation/package.py#L53

Relevant line in docs:
https://awscli.amazonaws.com/v2/documentation/api/latest/reference/cloudformation/deploy/index.html#options:~:text=%2D%2D-,template%2Dfile,-(string)%20The%20path

**Comment by tim-finnigan** [comment, marked as answer]
HI @qayshp thanks for the suggestion. You're welcome to create a PR for wording suggestions like this to get more feedback.  Typically documentation models are owned upstream by service teams and imported separately but customizations like this are an exception. Though we'd still want to confirm any changes with the CloudFormation team. Are you not able to access the template file via a relative path?
